aliyun/src/test/java/org/apache/iceberg/aliyun/oss/AliyunOSSTestUtility.java

```diff
@@ -0,0 +1,58 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.apache.iceberg.aliyun.oss;
+
+import org.apache.iceberg.aliyun.oss.mock.AliyunOSSMockRule;
+import org.apache.iceberg.common.DynConstructors;
+import org.apache.iceberg.relocated.com.google.common.base.Strings;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class AliyunOSSTestUtility {
+  private static final Logger LOG = LoggerFactory.getLogger(AliyunOSSTestUtility.class);
+  private static final String ALIYUN_OSS_TEST_RULE_CLASS = "ALIYUN_OSS_TEST_RULE_CLASS";
```

Comment:
In our own test environments, all the env variables has the same prefix `ALIYUN_TEST_` because people could easily identify them from the common prefix:

```bash
export ALIYUN_TEST_OSS_TEST_RULE_CLASS=org.apache.iceberg.aliyun.oss.OSSIntegrationTestRule
export ALIYUN_TEST_BUCKET=iceberg-test                                                                                     
export ALIYUN_TEST_ACCESS_KEY_ID=******                        
export ALIYUN_TEST_ACCESS_KEY_SECRET=******              
export ALIYUN_TEST_OSS_WAREHOUSE=******
export ALIYUN_TEST_OSS_ENDPOINT=******               
export ALIYUN_TEST_DLF_CATALOG_ID=******                               
export ALIYUN_TEST_DLF_END_POINT=******                 
export ALIYUN_TEST_DLF_REGION_ID=******
```

I will suggest use the same prefix.